lib/senders/sms.js

```diff
@@ -96,13 +97,13 @@ module.exports = (log, translator, templates, config) => {
         }
 
         const now = new Date()
-        const minuteAgo = new Date(now.getTime() - MILLISECONDS_PER_MINUTE)
+        const minuteAgo = new Date(now.getTime() - PERIOD_IN_MINUTES * MILLISECONDS_PER_MINUTE)
```

Comment:
Tiny nit, but `minuteAgo` is a misleading name for this now. Either `fiveMinutesAgo`, or something along the lines of `startOfPeriod` if we don't want to hardcode the detail in the name, might be better?

Reply:
good point